### PR TITLE
Added a CrateDB mimetype, instead of using plsql

### PR DESCRIPTION
**Pull request by Autophagy**
partitioned

### CODE REVIEW

File: app/tests/controllers/console.test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  });
+
+  describe('KeywordObjectCreator', function() {
+    it('should create object from keyword array', inject(function() {
+      var keywordArray = ['select', 'from', 'limit'];
+      var keywordsObject = KeywordObjectCreator.create(keywordArray);
+      expect(keywordsObject).toEqual({'select': true, 'from': true, 'limit': true});
+    }));
+  });
+
```
Comment (chaudum) [inline app/tests/controllers/console.test.js]: optional: add another test that checks what happens if you have duplicate keywords in the initial array
